torch/testing/__init__.py

```diff
@@ -197,13 +210,23 @@ def _helper(a, b, s) -> _compare_return_type:
         a = complex(a)
         b = complex(b)
 
+        if equal_nan == "relaxed":
+            if math.isnan(a.real) or math.isnan(a.imag):
```

Comment:
use cmath.isnan(a) and cmath.isnan(b) for readability

Reply:
fixed